README.md

```diff
@@ -142,10 +142,10 @@ This repository also contains the source code for the VS Code extension in the
 folder.
 It is also available via the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=VOSCA.vscode-v-analyzer).
 
-## NVIM LSP
+## NVIM LSP / Mason
```

Comment:
What is Mason in this context?

Reply:
If it is https://mason.app/ I wouldn't bother merging this change, as they are shutting down support.  Check the page.

Looks like it might've been nifty, billed as an "AI powered SQL editor", but...

Reply:
> What is Mason in this context?

That’s this one: https://github.com/williamboman/mason.nvim, it is also a very popular LSP installer

Reply:
Thank you.